src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,7 +26,7 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<T> machines, T value);
+    void fill(List<? super Machine> machines, T value);
```

Comment:
```suggestion
    void fill(List<? super T> machines, T value);
``` сорі за неправильну корекцію, сам на те попався, але якщо в тебе інтерфейс параметризований <T extends Machine>, то методи параметризуються відносно T